src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.Command_contract_has_not_been_broken.approved.AddPackage.json

```diff
@@ -1 +1 @@
-{"token":"the-token","commandType":"AddPackage","command":{"packageReference":{"packageName":"MyAwesomePackage","packageVersion":"1.2.3","isPackageVersionSpecified":true},"targetKernelName":null,"id":"command-id"}}
\ No newline at end of file
+{"token":"the-token","commandId":"command-id","commandType":"AddPackage","command":{"packageReference":{"packageName":"MyAwesomePackage","packageVersion":"1.2.3","isPackageVersionSpecified":true},"targetKernelName":null}}
```

Comment:
The meaning of "token" and "command-id" is going to be pretty unclear to people, I think. We should discuss.

Reply:
done